src/rgw/rgw_rados.h

```diff
@@ -2302,7 +2302,8 @@ class RGWRados : public AdminSocketHook
 
   int process_lc();
   int list_lc_progress(const string& marker, uint32_t max_entries, map<string, int> *progress_map);
-  
+  int lc_get_entry(const string& lc_oid, string& marker, pair<string, int>& entry);
+
```

Comment:
rgw::lc::fix_lc_shard_entry() is calling cls_rgw_lc_get_entry() directly, so i don't think we need this helper function in RGWRados

Reply:
Good catch, I thought I dropped this during rebase, dropping.